src/Emulator/Visuals/Visuals.css

```diff
@@ -2,4 +2,62 @@
     background-color: darkgrey;
     height: 50vh;
     width: 75vw;
-}
\ No newline at end of file
+}
+
+.imageContainer {
```

Comment:
I like modules. :)
My understanding that it prevents naming collisions between imported CSS files, is that correct? Happy to help with that.

Reply:
I like xx-xxx-x

Reply:
@takakonishimura Yes! It scopes class names to only the file that it is imported in so that there are no naming collisions.
I think that if we do want to move towards that, camel case is best for CSS names because that's what you use for CSS modules